tests/Unit/FilesystemTestCase.php

```diff
@@ -7,43 +7,71 @@
 
 abstract class FilesystemTestCase extends VirtualFilesystemTestCase {
 	protected $wprocket_structure = [
```

Comment:
@Tabrisrp Change this to `$structure`. Then it will override the [base structure found here]()https://github.com/wp-media/phpunit/blob/master/Unit/VirtualFilesystemTestCase.php#L22-L30.